source/source-connector/usage-examples/schema.txt

```diff
@@ -140,7 +140,7 @@ Your custom schema connector configuration should resemble the following:
 
 .. note:: Embedded Schema
 
-   In the preceding configuration, the JSON Schema converter embeds the custom
+   In the preceding configuration, the {+json-schema-converter+} embeds the custom
    schema in your messages. To learn more about the JSON Schema converter, see the
```

Comment:
Q: Are we only capitalizing "converter" when using the full product name? There are several places on this page, like this one, where it isn't capitalized.

Reply:
Correct - the official recommendation was to only capitalize it during the full product name. I opted to leave JSON Schema in for clarity purposes.